hazelcast/src/main/java/com/hazelcast/replicatedmap/impl/ReplicatedMapService.java

```diff
@@ -295,7 +295,7 @@ public ReplicatedRecordStore getReplicatedRecordStore(String name, boolean creat
     }
 
     public void initializeListeners(String name) {
-        List<ListenerConfig> listenerConfigs = config.getReplicatedMapConfig(name).getListenerConfigs();
+        List<ListenerConfig> listenerConfigs = getReplicatedMapConfig(name).getListenerConfigs();
         for (ListenerConfig listenerConfig : listenerConfigs) {
```

Comment:
If `listenerConfigs` is `null` (which seems to be allowed), this `for each` will throw a NPE, correct?

Reply:
`getListenerConfigs()` creates and returns an empty `ArrayList` if `listenerConfigs` is found to be `null`.